### PR TITLE
Security space suit

### PR DESCRIPTION
**Pull request by MatheusIsNotAvailable**
[FEATURE] [ADD TO WIKI]

## About the PR 

This PR adds a space suit to the security equipment locker.
Captura_de_Tela_92

## Why's this needed? 

Sec officers lack protection from space, wich makes space arrests or chases in space more complicated than what they should be for situations that happen so frequently.

## Changelog

```
(u)Arthur Holiday:
(*)Added security space suits to sec podbays.
```

### CODE REVIEW

**Comment by UrsulaMejor** [comment]
Sec officers lack protection from space because they are not meant to have protection from space. Ask the heads to open EVA, or use the security pods + that one attachment if forget the name of that kidnaps people into your pod

**Comment by Flaborized** [comment]
Don't like the idea, think it's unneeded and that EVA is Fully Enough, plus the pods. The meta would likely be to keep a full suit in your bag which I don't like. I also don't really like the sprites, especially the highlights on the helmet and the side sprites for the helemts, looks Weird to me

**Comment by Emcee-Gore** [comment]
> 
> 
> Sec officers lack protection from space because they are not meant to have protection from space. Ask the heads to open EVA, or use the security pods + that one attachment if forget the name of that kidnaps people into your pod

Why do they have security ocean suits then? The trench is their space equivalent.

File: code/obj/storage/secure_closets.dm [hunk 1]
```diff
@@ -175,6 +175,8 @@
 	name = "\improper Security equipment locker"
 	spawn_contents = list(/obj/item/storage/box/clothing/security,
 	/obj/item/clothing/suit/wintercoat/security,
+	/obj/item/clothing/head/helmet/space/security,
```
Comment (Emcee-Gore) [inline code/obj/storage/secure_closets.dm:178]: There is no reason for the space-suit to be on the ocean maps.

I recommend adding these suits manually to each map-file or finding a storage object that you can only find on the space-maps to modify.

Comment (MatheusIsNotAvailable) [reply code/obj/storage/secure_closets.dm:178]: That is true, I should do that.

**Comment by UrsulaMejor** [comment]
> > Sec officers lack protection from space because they are not meant to have protection from space. Ask the heads to open EVA, or use the security pods + that one attachment if forget the name of that kidnaps people into your pod
> 
> Why do they have security ocean suits then? The trench is their space equivalent.

the ocean is not equivalent to space, and ocean maps play differently to space maps. notably, all you need to go into the ocean is internals, and so the diving suit is dramatically more niche than a space suit is

**Comment by Emcee-Gore** [comment]
> 
> 
> > > Sec officers lack protection from space because they are not meant to have protection from space. Ask the heads to open EVA, or use the security pods + that one attachment if forget the name of that kidnaps people into your pod
> > 
> > 
> > Why do they have security ocean suits then? The trench is their space equivalent.
> 
> the ocean is not equivalent to space, and ocean maps play differently to space maps. notably, all you need to go into the ocean is internals, and so the diving suit is dramatically more niche than a space suit is

Mind that I mentioned the trench, not the ocean surrounding the station.

The trench does inflict mad burn dmg unless one wears proper heat-protection-gear like the security diving suit.
Similar to how space does inflict mad burn dmg unless one wears a proper space-wear suit.

**Comment by UrsulaMejor** [comment]
the "surrounding the station" area is the only part that's relevant to sec 99% of the time. security benefits from going to the trench basically never, unless it's to recover a body, but access to the local space outside cog 1 to seamlessly chase down changelings and the like would always be relevant

**Comment by Emcee-Gore** [comment]
> 
> 
> the "surrounding the station" area is the only part that's relevant to sec 99% of the time. security benefits from going to the trench basically never, unless it's to recover a body, but access to the local space outside cog 1 to seamlessly chase down changelings and the like would always be relevant

Fair.

Well - then perhaps this suit would most appropriately fit in as a job-reward or achievement reward to re-skin the space-suit so that it looks like a security one?

**Comment by MatheusIsNotAvailable** [comment]
If the PR gets rejected I will add it as a sec officer job reward and come up with some ways for sec officers to get job xp, or maybe a medal reward.

**Pull request closed without merging**